modules/ROOT/pages/custom-rulesets-new.adoc

```diff
@@ -1,6 +1,6 @@
 = Creating Completely New Custom Rulesets
 
-If you want a custom ruleset that cannot be created by modifying one of the provided rulesets, use one of the following approaches:
+If you want a custom ruleset that you can't create by modifying one of the provided rulesets, use one of the following approaches:
```

Comment:
```suggestion
If you can't create a custom ruleset by modifying one of the provided rulesets, use one of the following approaches:
```

Reply:
This reword would be technically misleading, since they can always create one, but they might not want to. I'll think about rewording to make that more clear.

Reply:
I've reworded it as follows:

To create a completely new custom ruleset that isn't based on an existing one ...